tests/network/probes.go

```diff
@@ -227,6 +228,28 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			Entry("[test_id:TODO]with working Exec probe", createExecProbe(period, initialSeconds, timeoutSeconds, "uname", "-a"), blankIPFamily),
 		)
 
+		Context("guest agent ping", func() {
+			BeforeEach(func() {
+				vmi = libvmi.NewFedora(withMasqueradeNetworkingAndFurtherUserConfig(
+					withLivelinessProbe(createGuestAgentPingProbe(period, initialSeconds)),
+				)...)
+				vmi = tests.VMILauncherIgnoreWarnings(virtClient)(vmi)
```

Comment:
nit: could we inline these calls ? 

feel free to disregard.

Reply:
I want to keep the vmi object creation and adding it to the cluster separate.

Reply:
I agree that might be useful when debugging - something I would accept for a unit test.

Anyway, your call - I won't block over this.

Reply:
Do we really need `VMILauncherIgnoreWarnings` ? Can't we use the regular func to launcher the VMI ?

Reply:
I copied the code from other test.
Ok, I've changed it to `tests.RunVMIAndExpectLaunchIgnoreWarnings()`.

Reply:
Ok, I will change it. I don't know why the other tests ignore warnings.

Reply:
Thank you; please let us know what warnings are thrown if this happens to fail, and we can evaluate if those warnings are reasonable or not.